app/experimenter/nimbus-ui/src/components/PageEditAudience/FormAudience/index.tsx

```diff
@@ -107,10 +107,19 @@ export const FormAudience = ({
     () =>
       [false, true].map((next) =>
         handleSubmit(
-          (dataIn: DefaultValues) => !isLoading && onSubmit(dataIn, next),
+          (dataIn: DefaultValues) =>
+            !isLoading &&
+            onSubmit(
+              {
+                ...dataIn,
+                locales,
+                countries,
```

Comment:
I would have thought `countries` and `locales` would appear in the `dataIn` object automatically because they are registered components. But alas, this is not happening, and is even how we do it for the Metrics page.